code-challenges.js

```diff
@@ -7,16 +7,45 @@ var secretCodeWord1 = "lackadaisical"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
 
+const l33tSp34k = (string) => {
+    tempArr = string.split("")
+    return tempArr.map(value => {
+        if (value === "a") {
+            return value.replace(/a/gi, "4")
+        } else if (value === "e"){
+            return value.replace(/e/gi, "3")
+        } else if (value === "i"){
+            return value.replace(/i/gi, "1")
+        } else if (value === "o"){
+            return value.replace(/o/gi, "0")
```

Comment:
Nice use of Regex! I would encourage you to make sure you comment how it works for those who may not be as familiar with the usage here.